crates/re_viewport/src/space_view_heuristics.rs

```diff
@@ -165,6 +171,9 @@ fn is_interesting_space_view_not_at_root(
     classes_with_interesting_roots: &[SpaceViewClassName],
     query: &LatestAtQuery,
 ) -> bool {
+    // TODO(jleibs): Why not make everything interesting?
+    return true;
```

Comment:
Much preferred. The whole idea of not having space_view at root be always a thing is from back then when we never allowed it in the first place.

The heuristics try to allow two ways of thinking _and a mix of it_ right now: One where you have one scene at the root and one where you have many scenes each under a `/thing`. Back in the day we _only_ had the latter.